src/Nethermind/Nethermind.WebSockets/ReceiveResult.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.WebSockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nethermind.WebSockets
+{
+    public class ReceiveResult
+    {
+        public int Read { get; set; }
+        public bool EndOfMessage { get; set; }
+        public bool Closed { get; set; }
+        public WebSocketCloseStatus? CloseStatus { get; set; }
```

Comment:
Should we have WebSocketReceiveResult : ReceiveResult with the WebSocket only properties?

Reply:
Agree it doesn't look perfect but I didn't want to interfere with the existing logic to much.
I'll try to improve it.